### DISCUSSION TITLE
Updating edirect formula

### DISCUSSION THREAD

**Discussion by nucholab** [brewsci/homebrew-bio#1367] [Q&A]
The recent versions of edirect have some big changes. I would like to update the formula to use the newer version(s) and to make sure it can be installed and run on Apple Silicon. Could someone point me to good educational resources or tips on how to go about this (I'm a novice at writing formulae).

Thanks!

Wladimir

**Comment by sjackman** [comment]
That'd be great! Thanks, Wladimir! See https://docs.brew.sh/#contributors

GitHub Actions does not yet have support for Apple M1, so we can't build a binary package (called a bottle) for Apple M1, but I'm happy to accept changes to the formulae to allow building from source on Apple M1.

See https://github.com/actions/virtual-environments/issues/2187#issuecomment-737339211
